lbz/authentication.py

```diff
@@ -1,14 +1,11 @@
 """
 JWT based Authentication module.
 """
-import os
-
+from lbz._cfg import REMOVE_PREFIXES
```

Comment:
Previously, it was a boolean value but now here, you get the `EnvValue` object - you need to use its `.value` to work with it correctly below.